symbolic-symcache/tests/snapshots/test_cache__lookup.snap

```diff
@@ -3,59 +3,19 @@ source: symbolic-symcache/tests/test_cache.rs
 expression: "&line_infos"
```

Comment:
this snapshot is pretty useless now, as it prints the `SourceLocationIter` and not the actual source locations

Reply:
Fair point. I actually fixed that in a branch somewhere, let me find it.